### ISSUE TITLE
Help pushing a new repository

### ISSUE THREAD

**Issue by dvdmon** [jjaimealeman/wdb-get-good-with-git#28]
So this isn't specific to this repository, but I recently decided I should put my YelpCamp project on github so that I can work on it from multiple PCs.  I followed instructions, but am getting a very unhelpful vague error and was hoping someone might clue me in as to what I'm doing wrong.  I'm currently logged into github on the PC I'm doing this on FYI: 
![image](https://user-images.githubusercontent.com/10165620/144717361-2e0a242c-1601-4e2f-bc12-64d5d9381054.png)

**Comment by dvdmon** [comment]
Never mind, realized that I needed to replace git@github.com with my actual github userid!

**Comment by jjaimealeman** [comment]
@dvdmon 
lol
This happens to me all the time.
I post some help to some community and within minutes, I solve it myself. hahaha.

Glad it was an easy fix buddy!

**Comment by dvdmon** [comment]
Actually, I was able to commit and push and set my remote to dvdmon@github.com:dvdmon/YelpCamp.git.  However when I go on GitHub, that repository is still empty... ??

**Comment by jjaimealeman** [comment]
@dvdmon 
Doesnt exist https://github.com/dvdmon?tab=repositories
Gotta create first

**Comment by dvdmon** [comment]
Yeah, I did.  Ugh, stupid SSH again, getting "Permission denied (publickey)"

**Comment by dvdmon** [comment]
Ok, figured it out - it SHOULD have been git@github.com.  I just hadn't yet added and committed my files initially.
